solutions/usaco-1088.mdx

```diff
@@ -11,3 +11,40 @@ author: Melody Yu (Video), Tyler Xiao (Video)
 
 Note: The video solution might not be the same as other solutions. Code in Java.
 <Youtube id="7eMHxpDQJEw" />
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N^2)$
+
+<LanguageSection>
+
+<PySection>
+
+```py
+n = int(input())
+# grid_by_rows[i] is the ith row
+grid_by_rows = [list(map(int, input().split())) for _ in range(n)]
+# grid_by_cols[i] is the ith column
+grid_by_cols = zip(*grid_by_rows)
+
+rows_alternate, cols_alternate = 0, 0
```

Comment:
```suggestion
rows_alternate = 0
cols_alternate = 0
```

Reply:
also if you're gonna add some comments might as well add some explaining these variables

Reply:
> also if you're gonna add some comments might as well add some explaining these variables